lib/omniauth/strategies/younility.rb

```diff
@@ -8,8 +8,8 @@ class Younility < OmniAuth::Strategies::OAuth2
 
       option :client_options, {
         site:           'https://app.younility.com',
```

Comment:
this may still bite you (or are you able to change this in the web app to, e.g., `staging-api.younility.com`?)